Doc/library/asyncio-task.rst

```diff
@@ -328,6 +328,13 @@ Running Tasks Concurrently
    If ``gather()`` is *cancelled*, all submitted awaitables
    (that have not completed yet) are also *cancelled*.
 
+   If *return_exceptions* is False, cancelling gather() after it
```

Comment:
I'd put this into a `.. note::` section just to make it easier for the reader visually. And shift it below the example.  The text is good.

Reply:
done